### PR TITLE
docs(api-request-builder): documentation improvements

### PR DESCRIPTION
**Pull request by jherey**
#### Summary

additions to api-request-builder documentation

#### Description

display API endpoints, services and features in docs

### CODE REVIEW

File: docs/sdk/api/apiRequestBuilder.md [hunk 1]
```diff
@@ -204,6 +274,9 @@ This declarative `parse` API accepts an object of the following shape:
 
   // dataErasure
   dataErasure?: string;
+
+  // params
+  orderNumber?: number,
```
Comment (daern91) [inline docs/sdk/api/apiRequestBuilder.md:279]: 🙏

Comment (jherey) [reply docs/sdk/api/apiRequestBuilder.md]: Changed it to a semi-colon. Thanks for the catch

Comment (daern91) [reply docs/sdk/api/apiRequestBuilder.md]: Hah, I honestly didn't even see it. Just thought it was nice that you added it since we forgot in the last PR.

Comment (jherey) [reply docs/sdk/api/apiRequestBuilder.md]: Oh wow...nice. When I see Daniel's comment, I have to double-check 😆

**Pull request merged**